applications/vessel-history/src/features/dataviews/dataviews.slice.ts

```diff
@@ -62,6 +62,18 @@ export const selectDataviewById = memoize((id: number) =>
   createSelector([(state: RootState) => state], (state) => selectById(state, id))
 )
 
-export const selectDataviewsStatus = (state: RootState) => state.dataviews.status
+export const selectDataviews = createSelector(
+  [(state: RootState) => state.dataviews],
+  (dataviews) => {
+    return dataviews
+  }
+)
+
+export const selectDataviewsStatus = createSelector(
+  [selectDataviews],
+  (dataviews) => {
+    return dataviews.status
+  }
+)
```

Comment:
I'd revert these changes, the problem was not here and having two selectors to consume only one `selectDataviewsStatus` from other places is not ideal.